### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by aksoli666**
+added .env, docker-compose.yml, Dockerfile

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,19 +21,23 @@
...
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>3.1.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
+			<version>3.1.0</version>
```
Comment (MentorMA) [inline pom.xml:40]: write in properties and reuse this value

File: pom.xml [hunk 2]
```diff
@@ -43,37 +47,38 @@
 		<dependency>
 			<groupId>org.liquibase</groupId>
 			<artifactId>liquibase-core</artifactId>
+			<version>4.21.0</version>
 		</dependency>
 		<dependency>
 			<groupId>org.liquibase</groupId>
 			<artifactId>liquibase-maven-plugin</artifactId>
-			<version>4.22.0</version>
+			<version>4.21.0</version>
```
Comment (MentorMA) [inline pom.xml:55]: The same

File: src/main/java/online/book/store/model/Order.java [hunk 3]
```diff
@@ -52,6 +53,7 @@ public class Order {
     @Column(nullable = false)
     private BigDecimal total;
 
+    @CreationTimestamp
     @Column(name = "order_date", nullable = false)
     private LocalDateTime orderDate;
```
Comment (MentorMA) [inline src/main/java/online/book/store/model/Order.java:58]: This changes not suited with docker task

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/online/book/store/model/Order.java (hunk 3 above)
Comment (MAMentorFX) [reply src/main/java/online/book/store/model/Order.java:58]: not fixed

**Pull request merged**